package.json

```diff
@@ -27,6 +27,7 @@
     "test:vite-ci": "turbo run test --filter=astro",
     "test:e2e": "cd packages/astro && pnpm playwright install && pnpm run test:e2e",
     "test:e2e:match": "cd packages/astro && pnpm playwright install && pnpm run test:e2e:match",
+    "test:e2e:hosts": "turbo run test:hosted",
```

Comment:
Since we're only running the tests weekly, do we want to use the turbo cache here? Otherwise we can use `pnpm run test:hosted -r` instead.

I'm also fine if we want to use turbo though.

Reply:
I did it this way so that it would run any future tests with the same name automatically, but yeah don't really need the cache